src/view/grants.njk

```diff
@@ -12,69 +12,68 @@
             <h1>The Trustlines Grants</h1>
             <h2>1. Background</h2>
             <p class="content-large">
-              The Trustlines Foundation pursues the charitable goal of promoting financial and economic inclusion of all people through decentralised peer-to-peer network protocols that serve common accounting. To ensure the pursuit of this goal in the long term, the Trustlines Foundation provides grants to projects that actively contribute to the development and the usage of the Trustlines ecosystem.
+              The Trustlines Foundation pursues the charitable goal of promoting financial and economic inclusion of all people through decentralized peer-to-peer network protocols that serve common accounting. To ensure the pursuit of this goal in the long term, the Trustlines Foundation provides grants to projects that actively contribute to the development and the usage of the Trustlines ecosystem.
             </p>
 
             <h2>2. Eligibility requirements</h2>
             <ul class="content-large">
-              <li><b>Open source software</b>
+              <li><b>Open-source software</b>
```

Comment:
it's Open Source, not open-source

Reply:
It is `open-source software`, or `open source` without the software.